core/platform/monitoring.go

```diff
@@ -0,0 +1,15 @@
+package platform
+
+// Observability keys
+const (
+	CapabilityIDKey        = "capabilityID"
+	TriggerIDKey           = "triggerID"
+	WorkflowIDKey          = "workflowID"
+	WorkflowExecutionIDKey = "workflowExecutionID"
+	WorkflowNameKey        = "workflowName"
+	WorkflowOwnerKey       = "workflowOwner"
+	StepIDKey              = "stepID"
+	StepRefKey             = "stepRef"
```

Comment:
Should these names have `Key` as a prefix instead of suffix? e.g. like std lib `package http`:
![image](https://github.com/user-attachments/assets/0a766901-246c-4082-9e38-aba6113cb56e)